src/assets/main.css

```diff
@@ -0,0 +1,106 @@
+* {
+  box-sizing: border-box;
+}
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Source Sans Pro', sans-serif;
+  font-weight: 400;
+}
+img {
+  max-width: 100%;
+}
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 0;
+  font-family: 'Playfair Display', serif;
+  font-weight: 700;
+}
+.container {
+  width: 95%;
+  max-width: 1180px;
+  margin: 0 auto;
+}
+footer {
+  text-align: left;
+  width: 100%;
+  background-color: #f4f4f4;
+  padding: 15px 2.5%;
+  margin-top: 15px;
+  font-size: 12px;
+}
+.grid {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-template-rows: auto;
+  grid-gap: 15px 15px;
+}
+@media (min-width: 450px) {
```

Comment:
there is a way to reuse this media inside a mixin to use exact numbers and avoid errors when this have to be repeated elseware, dig more into media mixins